db/schema.rb

```diff
@@ -820,6 +821,8 @@
 
   add_foreign_key "account_facility_joins", "accounts"
   add_foreign_key "account_facility_joins", "facilities"
+  add_foreign_key "account_users", "accounts", name: "fk_accounts"
```

Comment:
Are these two lines right?  I don't see a migration

Reply:
They alwas creep up because I think our schema.rb is not correct in all cases. I’ll take them out and address them separately, good catch!